pomodoro/src/main/java/pomodoro/dto/UserDto.java

```diff
@@ -1,10 +1,14 @@
 package pomodoro.dto;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class UserDto {
 
     private String email;
     private String firstName;
     private String lastName;
+    private List<PomodoroDto> pomodoros = new ArrayList<PomodoroDto>();
```

Comment:
Please use diamond operator instead of explicit type stating in the ArrayList creation.